src/components/DateRangeForm/DateRangeForm.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import Form from 'antd/lib/form';
+import Button from 'antd/lib/button';
+import DatePicker from 'antd/lib/date-picker';
+
+import './DateRangeForm.css';
+
+
+class DateRangeForm extends React.Component {
+    handleSubmit = e => {
+        e.preventDefault();
+
+        this.props.form.validateFields((err, fieldsValue) => {
+            if (err) {
+                console.log('error????');
+                return;
+            }
+            // Should format date value before submit.
+            const values = {
+                ...fieldsValue,
+                'date-time-picker-start': fieldsValue['date-time-picker-start'].format(
+                    'YYYY-MM-DDTHH:mm:ss'
+                ),
+                'date-time-picker-end': fieldsValue['date-time-picker-end'].format(
+                    'YYYY-MM-DDTHH:mm:ss'
+                ),
+            };
+            this.props.handleDateFilter(values);
+            this.props.handleDrawer(e);
+        });
+    };
+
+    handleClear = e => {
+        this.props.handleDateFilter(null);
+        this.props.handleDrawer(e);
+    };
+
+    render() {
+        const {getFieldDecorator} = this.props.form;
+        const formItemLayout = {
+            labelCol: {
+                xs: {span: 24},
+                sm: {span: 8},
+            },
+            wrapperCol: {
+                xs: {span: 24},
+                sm: {span: 16},
+            },
+        };
+        const config = {
+            rules: [
+                {type: 'object', required: true, message: 'Please select date!'},
+            ],
+        };
+        return (
+            <div className={'form-wrapper'}>
+                <Form {...formItemLayout} onSubmit={this.handleSubmit}>
+                    <div className={'date-wrapper'}>
```

Comment:
`className={'date-wrapper'}` can be simplified to `className="date-wrapper"`

Reply:
also probs wont need div for className with colours set by default now

Reply:
Oh some of the classNames are used for making the list in the rows stateful